schedule/yast/installer_extended/installer_extended@pvm.yaml

```diff
@@ -13,36 +13,22 @@ vars:
   EXIT_AFTER_START_INSTALL: '1'
   EXTRABOOTPARAMS: Y2STRICTTEXTDOMAIN=1
   YUI_REST_API: 1
-conditional_schedule:
-  access_beta_distribution:
-    BETA:
-      1:
-        - installation/validate_beta_message
-        - installation/access_beta_distribution
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - '{{access_beta_distribution}}'
-  - installation/product_selection/install_SLES
-  - installation/licensing/verify_license_translations
-  - installation/licensing/verify_license_has_to_be_accepted
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/view_development_versions
-  - installation/module_registration/verify_module_registration
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/accept_proposed_layout
-  - installation/releasenotes
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/default_user_simple_pwd
-  - installation/authentication/root_simple_pwd
-  - installation/installation_settings/validate_ssh_service_enabled
-  - installation/installation_settings/open_ssh_port
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
+  license_agreement:
+    - installation/licensing/verify_license_translations
+    - installation/licensing/verify_license_has_to_be_accepted
+    - installation/licensing/accept_license
+  registration:
+    - installation/registration/register_via_scc
+    - installation/module_registration/view_development_versions
+    - installation/module_registration/verify_module_registration
```

Comment:
should be under `extension_module_selection`

Reply:
done.